tools/n2to3co2/n2to3co2.f

```diff
@@ -0,0 +1,742 @@
+c----------------------------------------------------------------------
+      program n2to3co2
```

Comment:
Why do we need n2to3co2 and the regular n2to3?

Reply:
n2to3co2 is co2 generator using i16. Keeping this tool separate from n2to3 is simpler instead of putting two functions into one tool, n2to3. At least, trying to keep 1B support version to be as minimal change as possible without much deviating from previous version in terms of format and style.

Reply:
n2to3co2 is more or less a copy of n2to3 with a minor extension. I prefer to avoid any kind of code duplication unless there is a very good reason to do so. For now I am ok to have an extra version. Medium term co2 should be replaced by parCon anyway.